nextflow.config

```diff
@@ -103,9 +119,10 @@ params {
     igenomes_ignore            = false
     validate_params            = true
     show_hidden_params         = false
-    schema_ignore_params       = 'genomes,callers,dedup_control_only,run_igv,run_peak_plotting,run_multiqc,run_deep_tools,run_reporting,run_consensus_all,run_peak_calling,run_remove_dups,run_mark_dups,run_q_filter,run_alignment,run_trim_galore_fastqc,run_cat_fastq,run_input_check,run_genome_prep'
+    schema_ignore_params       = 'genomes,callers,dedup_control_only,fragment_size,run_igv,run_multiqc,run_reporting,run_consensus_all,run_peak_calling,run_remove_dups,run_mark_dups,run_read_filter,run_alignment,run_trim_galore_fastqc,run_cat_fastq,run_input_check,run_genome_prep,run_peak_qc,run_deeptools_qc,run_deeptools_heatmaps,run_preseq'
```

Comment:
You seem to have a lot of params that are ignored from the schema, any reason for that?

Reply:
yep, these are internal flow parameters that shouldn't be exposed to the user. The flow params are `skip` and `only` - these combinations plus other params determines what actually runs using switching on `run`

Reply:
shouldn't it be easier to deal with that with default values in the schema?